android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -290,6 +299,58 @@ public void captureEnvelope(ReadableArray rawBytes, ReadableMap options, Promise
         promise.resolve(true);
     }
 
+    @ReactMethod
+    public void captureScreenshot(Promise promise) {
+        final Activity activity = this.getReactApplicationContext().getCurrentActivity();
+        if (activity == null
```

Comment:
Consider this check as well https://github.com/getsentry/sentry-java/blob/38e4f116623e0dd1329f54d86dd4bbe5dd687a86/sentry-android-core/src/main/java/io/sentry/android/core/ScreenshotEventProcessor.java#L182-L186